src/sentry/discover/endpoints/discover_key_transactions.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import absolute_import
+
+from django.db import transaction
+from rest_framework.response import Response
+from rest_framework.exceptions import ParseError
+
+from sentry import features
+from sentry.api.bases import OrganizationEventsV2EndpointBase
+from sentry.api.bases.organization import OrganizationPermission
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.discover.models import KeyTransaction
+from sentry.discover.endpoints.serializers import KeyTransactionSerializer
+from sentry.snuba.discover import query
+
+
+class KeyTransactionEndpoint(OrganizationEventsV2EndpointBase):
+    permission_classes = (OrganizationPermission,)
+
+    def has_feature(self, request, organization):
+        return features.has("organizations:performance-view", organization, actor=request.user)
+
+    def get_project(self, request, organization):
+        projects = self.get_projects(request, organization)
+
+        if len(projects) != 1:
+            raise ParseError("Only 1 project per Key Transaction")
+        return projects[0]
+
+    def post(self, request, organization):
+        """ Create a Key Transaction """
+        if not self.has_feature(request, organization):
+            return Response(status=404)
+
+        project = self.get_project(request, organization)
+
+        base_filter = {"organization": organization, "project": project, "owner": request.user}
+
+        with transaction.atomic():
+            serializer = KeyTransactionSerializer(data=request.data, context=base_filter)
+            if serializer.is_valid():
+                data = serializer.validated_data
+                base_filter["transaction"] = data["transaction"]
+
+                KeyTransaction.objects.create(**base_filter)
+                return Response(status=201)
+            else:
+                return Response(serializer.errors, status=400)
```

Comment:
```suggestion
            return Response(serializer.errors, status=400)
```
Don't need an else when the if returns 🏌